Domains/2-DisciplinePhysical/StructuralPhysical/StructuralPhysical.ecschema.xml

```diff
@@ -73,7 +73,7 @@
             <Class class="WallType"/>
         </Target>
     </ECRelationshipClass>
-    <ECEntityClass typeName="Beam" displayLabel="Beam">
+    <ECEntityClass typeName="Beam" displayLabel="Beam" description="A bis:StructuralMember; one-dimensional horizontal structural element used in building construction to support and transfer loads to columns or walls.">
         <BaseClass>StructuralMember</BaseClass>
     </ECEntityClass>
     <ECEntityClass typeName="BeamType" displayLabel="Beam Type">
```

Comment:
When it gets tricky to provide a specialized description for a PhysicalType, we've been using a generic description worded like "Defines a shared set of properties whose values vary per-type of <physical-element class name - e.g. Beam> rather than per-instance.".